### PR TITLE
docs: Adapt get-started & common guides to use OCI format for Helm

### PR DESCRIPTION
**Pull request by juan131**
## Summary 

This PR adapts the "get started" & common guides to use the OCI format for Helm.

### CODE REVIEW

**Review by carrodher — COMMENTED** [review]
Thanks for the PR! In the end, we shouldn't have any mention of `helm repo add`.

It seems there is also a mention at _common/_install_tac.md.erb_
```
18:        $ helm repo add <%= variable :repository_name, :platform %> REPOSITORY --username "USERNAME" --password "PASSWORD"
```

File: _get_started_kubernetes.md.erb [hunk 1]
```diff
@@ -131,7 +131,7 @@ To run your commands against Kubernetes clusters, the *kubectl* CLI is needed. C
 
 <%= partial 'platforms/kubernetes/install_configure_helm.md.erb' %>
 
-Add the Bitnami repository to Helm with the following command:
+Add the Bitnami repository to Helm with the following command so you can later on discover charts and available versions:
 
     $ helm repo add bitnami https://charts.bitnami.com/bitnami
```
[the remark this answers is not in the document]
Comment (juan131) [reply _get_started_kubernetes.md.erb:136]: idem

**Comment by juan131** [comment]
>  there are some commands installing Helm charts using the previously added repository, see the whole list at (...)

According to Beltran Rueda, the changes on the specific app's section aren't required since we'll be replacing these sections with README(s).

**Comment by carrodher** [comment]
> > there are some commands installing Helm charts using the previously added repository, see the whole list at (...)
> 
> According to Beltran Rueda, the changes on the specific app's section aren't required since we'll be replacing these sections with README(s).

It's true we are going to only keep the README as the source of truth for documentation, the thing is that there are some sections that were moved to chart-docs and linked from the README that we'll need to move back again from chart-docs to the README.
Although being said that, it is very likely that the mentioned sections we will need to move do not contain this kind of instruction. We can keep them as they are and modify them if needed when doing the movement.

File: common/_install_kubernetes.md.erb [hunk 2]
```diff
@@ -11,9 +11,9 @@ highlight: 20
 
 > NOTE: To install a Helm chart repository, it is necessary to have [Helm previously installed and configured in your cluster](/<%=platform_path %>/faq/get-started/install-configure-helm).
 
-To start using any <%= variable :catalog_name, :platform %> Helm chart, it is necessary to first add the <%= variable :catalog_name, :platform %> Helm chart repository to Helm and then run the *helm install* command to deploy this chart. Follow these steps:
+To start using any <%= variable :catalog_name, :platform %> Helm chart, follow these steps:
 
-* Add the <%= variable :catalog_name, :platform %> repository to Helm with the following command:
+* Add the <%= variable :catalog_name, :platform %> repository to Helm so you can later on discover charts and available versions:
 
         $ helm repo add bitnami https://charts.bitnami.com/bitnami
```
[the remark this answers is not in the document]
Comment (juan131) [reply common/_install_kubernetes.md.erb]: done at https://github.com/bitnami/charts-docs/pull/149/commits/dca6d95b0813c46af3967a0bb7bc06f44f3f8517

**Review by carrodher — APPROVED** [review]
LGTM!

**Pull request merged**